cli/src/main/java/com/devonfw/tools/ide/context/AbstractIdeContext.java

```diff
@@ -599,7 +629,9 @@ public GitContext getGitContext() {
   @Override
   public ProcessContext newProcess() {
 
-    return new ProcessContextImpl(this);
+    ProcessContext processContext = new ProcessContextImpl(this);
+    processContext.directory(this.getDefaultExecutionDirectory());
```

Comment:
The default here is `null`. Don't you break the "productive" usage here?

Reply:
I would consider to change `getDefaultExecutionDirectory()` to return `Path.get(".")` (ideally from a constant) instead of `null`.